src/recon_buildblock/PLSPrior.cxx

```diff
@@ -105,23 +105,29 @@ PLSPrior<elemT>::post_processing()
 {
   if (base_type::post_processing()==true)
     return true;
-  if (kappa_filename.size() != 0)
-    this->kappa_ptr = read_from_file<DiscretisedDensity<3,elemT> >(kappa_filename);
 
-  if (anatomical_filename.size() != 0){
-    this->anatomical_sptr = read_from_file<DiscretisedDensity<3,elemT> >(anatomical_filename);
```

Comment:
shouldn't we read the anatomical image here as well?

Reply:
added in.